packages/Makefile

```diff
@@ -105,11 +109,21 @@ $(ANDROID_BUILD_DIR_MAV_CORE): $(ANDROID_CORE) $(MANIFEST_FILE)
 	$(UTIL_MAVEN_FORMAT_CORE_BIN) && touch $(ANDROID_BUILD_DIR_MAV_CORE)
 	@echo 'Done!'
 
-$(ANDROID_CORE): $(ANDROID_BUILD_DIR_INT_CORE) $(GO_SRC) $(GO_MOD_FILES) | $(GOMOBILE)
+$(ANDROID_CORE): $(ANDROID_BUILD_DIR_INT_CORE) $(GO_SRC) $(GO_MOD_FILES)
 	@echo '------------------------------------'
 	@echo '   Android Core: Gomobile binding   '
 	@echo '------------------------------------'
-	$(GOMOBILE) bind -v $(GOMOBILE_OPT) -target=android$(GOMOBILE_TARGET) -o $(ANDROID_CORE) $(CORE_PACKAGE) $(EXT_PACKAGE)
+	go mod download
+	go run golang.org/x/mobile/cmd/gomobile init
+	mkdir -p $(android_gomobile_cache) android/libs
+	GO111MODULE=on go run golang.org/x/mobile/cmd/gomobile bind \
+		-o $(ANDROID_CORE) \
+		-v $(GOMOBILE_OPT) \
+		-cache $(android_gomobile_cache) \
+		-target=android$(GOMOBILE_TARGET) \
```

Comment:
Can you copy this from Berty:
```
GOMOBILE_IOS_TARGET ?= ios
GOMOBILE_ANDROID_TARGET ?= android
```
and modify the rules using it accordingly?

Reply:
Done.